### ISSUE TITLE
Why python test_trajectories.py --settings_file=config/test_settings.yaml is so slow?

### ISSUE THREAD

**Issue by calmelo** [uzh-rpg/agile_autonomy#23]
When I run python test_trajectories.py --settings_file=config/test_settings.yaml, it is so slow to upload the model. I need to wait for several minutes. Can someone tell me the reason?

**Comment by ZhangHaley** [comment]
I have tried cuda11.4 it is more faster then other version.

**Comment by ZhangHaley** [comment]
I have tried CUDA11.4. It is faster than other versions.

**Comment by calmelo** [comment]
> I have tried CUDA11.4. It is faster than other versions.

The command of conda install tensorflow-gpu install the cuda10.1 default in tf_24 environment. How did you change the CUDA version？

**Comment by NPU-yuhang** [comment]
You can install tensorflow2.4 by pip instead of conda

**Comment by calmelo** [comment]
> You can install tensorflow2.4 by pip instead of conda

Did you mean that installing tensorflow2.4 by pip can install tensorflow2.4 with cuda11?

**Comment by NPU-yuhang** [comment]
Yes, but you should install cuda11 from Nvidia developer website before installing the tensorflow2.4.
